src/ussd_pmtct_seed.js

```diff
@@ -564,6 +539,55 @@ go.app = function() {
             });
         });
 
+        self.send_registration_thanks = function() {
+            var lang_code = self.im.user.lang || "eng_ZA";
+            var lang = engage.LANG_MAP[lang_code];
+            var content = self.im.user.i18n($(
+                "HIV positive moms can have an HIV negative baby! You can get free " +
+                "medicine at the clinic to protect your baby and improve your health"
+            ));
+            var text = self.im.user.i18n($(
```

Comment:
We can maybe rather name these something like `message1` and `message2`, so that it's a bit less confusing than `content` and `text`